addon.xml

```diff
@@ -8,7 +8,7 @@
 		<summary lang="en">Inter-addon signalling</summary>
 		<description lang="en">Provides signal/slot mechanism for inter-addon communication</description>
 		<platform>all</platform>
-		<license>GNU GENERAL PUBLIC LICENSE Version 2.1, February 1999</license>
+		<license>LGPL-2.1-only</license>
```

Comment:
While I'm not necessarily opposed to this change, I'm curious as to the reason for it.

Reply:
It is required in the future. In reviews of my add-ons I was asked to change this.

Reply:
https://github.com/xbmc/repo-scripts/pull/1213#pullrequestreview-321911606